google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/GcpCheckoutProvider.java

```diff
@@ -186,7 +186,8 @@ private static boolean doClone(@NotNull Project project, @NotNull ProgressIndica
       return true;
     }
     VcsNotifier.getInstance(project)
-        .notifyError(GctBundle.message("clonefromgcp.failed"), result.getErrorOutputAsHtmlString());
+        .notifyError(GctBundle.message("clonefromgcp.failed"), result.getErrorOutputAsHtmlString()
+            + " " + result.getOutputAsJoinedString());
```

Comment:
oops you're right, I don't know what I was looking out, getOutputAsHtmlString indeed does not exist